website/content/api-docs/secret/kv/kv-v1.mdx

```diff
@@ -116,11 +118,11 @@ policy granting the `update` capability.
 - `path` `(string: <required>)` – Specifies the path of the secrets to
   create/update. This is specified as part of the URL.
 
-- `:key` `(string: "")` – Specifies a key, paired with an associated value, to
+- `:key` `(string: "")` – Specifies a key in the payload, paired with an associated value, to
   be held at the given location. Multiple key/value pairs can be specified, and
   all will be returned on a read operation. A key called `ttl` will trigger
   some special behavior. See the [Vault KV secrets engine
-  documentation](/vault/docs/secrets/kv) for details.
+  documentation](/vault/docs/secrets/kv/kv-v1#ttls) for details.
```

Comment:
Previously, this linked back to the KV overview (e.g. explaining V1 vs V2) instead of the part of the docs that explains the TTLs.